ha/core/event_analyzer/parser/parser.py

```diff
@@ -191,6 +211,9 @@ def parse_event(self, msg: str) -> HealthEvent:
                 if specific_info and specific_info["generation_id"]:
                     generation_id = specific_info["generation_id"]
                     specific_info = {"generation_id": generation_id, "pod_restart": 0}
```

Comment:
Here also, we will need functional_type. Right?

Reply:
fixed